packages/sdk/src/domain/services/ConfigService.ts

```diff
@@ -39,4 +45,21 @@ export class ConfigService {
         return ConfigService.PROD_HELIO_SERVICE_BASE_URL;
     }
   }
+
+  getHelioBaseUrl(): string {
+    switch (this.cluster) {
+      case ClusterType.Testnet:
+      case ClusterType.Devnet:
+        return ConfigService.DEV_HELIO_BASE_URL;
+      case ClusterType.Mainnet:
+        return ConfigService.PROD_HELIO_BASE_URL;
+      default:
+        return ConfigService.PROD_HELIO_BASE_URL;
+    }
+  }
+
+  getPaymentFullLink(id: string): string {
```

Comment:
Ahh I see you put the ?ref here, is the variable name misleading? When do you use getPaymentFullLink other than phantom. Maybe can change to getPhantomLink